cuprum/unittests/test_extension_ci_contract.py

```
@@ -187,13 +57,29 @@ def test_the_ci_job_builds_the_extension_before_running_the_gated_tests() -> Non
         ('echo "make develop"', "make develop", False),
+        ("echo if make develop", "make develop", False),
+        ("cat <<EOF\nif make develop\nEOF", "make develop", False),
+        ("cat <<EOF\nmaturin develop\nEOF", "maturin develop", False),
+        (
+            "cat <<EOF\nmaturin develop\nEOF\nmaturin develop",
+            "maturin develop",
+            True,
+        ),
+        (
+            "cat <<FIRST <<SECOND\nignored first\nFIRST\nmaturin develop\n"
+            "SECOND\nmake develop",
```

Ruff (0.16.3)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)